### PR TITLE
Containerized deploy

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: buildJenkinsfile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+def deployConfigs
+
+pipeline {
+    agent {
+        node {
+            label 'team:iow'
+        }
+    }
+    parameters {
+        choice(choices: ['snapshots'], description: 'type of build', name: 'BUILD_TYPE')
```
Comment (mbucknell) [inline buildJenkinsfile:13]: Good that this is only snapshots. We shouldn't allow releases. I think we could arguably leave out any of the release code below.

File: pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+# This tells AWS what requests to redirect or route to this container
+urlRules:
+  - type: forward
+    conditions:
+      - "/monitoring-location/*"
+      - "/hydrological-unit/*"
+      - "/networks/*"
+      - "/states/*"
+      - "/components/*"
```
Comment (mbucknell) [inline pipeline.yml:42]: What is this one for?

File: buildJenkinsfile (hunk 1 above)
Comment (ayan-usgs) [reply buildJenkinsfile:13]: Removed the stage where tags are pushed back up to the repo.

File: pipeline.yml (hunk 2 above)
Comment (ayan-usgs) [reply pipeline.yml:42]: Isn't there an feature-toggled embeddable version of the plots? I guess this could be omitted. :thinking:

Comment (mbucknell) [reply pipeline.yml:42]: There is now that I look closer. Might as well leave in.

**Pull request merged**